ember-electron/test-main.js

```diff
@@ -14,23 +16,30 @@ let [, , indexUrl] = process.argv;
 indexUrl = indexUrl.replace(/__amp__/g, '&');
 console.log(`URL: ${indexUrl}`);
 let {
-  pathname: indexPath,
+  pathname: originalIndexPath,
   search: indexQuery,
 } = url.parse(indexUrl);
 // When we extract the pathname from an absolute path on windows, it starts
 // with '/C:/', and the leading slash confuses everything, so we need to strip
 // it.
 if (process.platform === 'win32') {
-  indexPath = indexPath.slice(1);
+  originalIndexPath = originalIndexPath.slice(1);
 }
-indexPath = resolve(indexPath);
+
+// Copy index-electron.html to a location not managed by broccoli so that it
+// doesn't get deleted when a change is detected.
+originalIndexPath = resolve(originalIndexPath);
+let emberAppDir = resolve(dirname(originalIndexPath), '..');
+let tmpDir = tmp.dirSync({ dir: resolve(emberAppDir, '../../'), unsafeCleanup: true }).name;
```

Comment:
Hmm...yeah, I guess the only time we use the system tmp location is when running the tests in this addon (i.e. not in a client app). But I think you're right that not having access to `tmp` is probably going to be fatal for other `electron`-related reasons...in any case we'll have to move the `tmp` module from `devDependencies` to `dependencies`.